### DISCUSSION TITLE
P.244 Loginインターフェースの記述ファイル

### DISCUSSION THREAD

**Discussion by manaty226** [budougumi0617/go_todo_app#27] [Ideas]
P.244の「`LoginService`インターフェースは`auth/service.go`に追記し，...」は「...`handler/service.go`に追記し...」が正しいかと思います．

**Comment by budougumi0617** [comment]
ご指摘ありがとうございました！正誤表に反映いたしました。
https://github.com/budougumi0617/go_todo_app/releases/tag/v1.0.3

**Reply by manaty226** [reply]
ご対応ありがとうございました！
